apidef/oas/middleware_test.go

```diff
@@ -175,6 +175,55 @@ func TestGlobal(t *testing.T) {
 	})
 }
 
+func TestTrafficLogs(t *testing.T) {
+	t.Parallel()
```

Comment:
t.Parallel is not necessary. We limit parallelization due to StartTest details, also it's only meant for slow tests. Data model conversions don't strike me as slow.

Please add some whitespace before assertions, consider readability; move `var` declarations to start of test functions to not intermix with logic that's being tested.